src/cpp/transport/custom/CustomAgent.cpp

```diff
@@ -198,27 +199,31 @@ bool CustomAgent::recv_message(
                 recv_endpoint_, buffer_, SERVER_BUFFER_SIZE, timeout, transport_rc);
         }
 
-        bool success = (0 <= recv_bytes && TransportRc::ok == transport_rc);
-        if (success)
+        if (TransportRc::ok == transport_rc)
```

Comment:
Your changes are the same as using:

```cpp
        bool success = (0 < recv_bytes && TransportRc::ok == transport_rc);
```

in the previous version?

Reply:
The only difference is that if recv_bytes=0= and transport_rc == OK (such as the case of fragmentation happening in the CRC due to the streaming protocol) the else if condition is not true and doesnt print the error message.